src/EnergyPlus/NodeInputManager.cc

```diff
@@ -1141,8 +1145,21 @@ namespace NodeInputManager {
 						}
 					}
 				}
+				if ( EMSManager::CheckIfNodeMoreInfoSensedByEMS( iNode, "System Node Wetbulb Temperature" ) ) { 
```

Comment:
I much prefer the full scope instead of using statements.  This goes against the majority of the code, but that is likely where we are going eventually...